python/shotgun_desktop/startup.py

```diff
@@ -92,9 +97,11 @@ def __desktop_engine_supports_authentication_module(engine):
 
     :returns: True if the engine supports the authentication module, False otherwise.
     """
-    if engine.version.lower() == 'undefined':
+    # Looking for v should be good enough. No need to complicate things with a regular
+    # expression. Github hashes start with a 0-f character.
```

Comment:
Fixes the issue with github hashes being lower than vX.Y.Z.

Reply:
Do we use a v prefix for github tag descriptors?? Do we have access to core at this point? Could we use the core util methods for version comparison?

Reply:
We do have access to core, but we are not guaranteed which version, hence we can't be guaranteed a tank.util.lower_version check would be available, if that's what you are thinking of. Over time it wouldn't be an issue however.

As for git tag descriptors, they use  the tag as is without any manipulation.